### DISCUSSION TITLE
Override "fill below" to fill up to a field matching a regex

### DISCUSSION THREAD

**Discussion by dominikwerder** [grafana/grafana#61596] [Dashboards, Panels & Visualizations]
The override "Fill below to" allows to choose a specific other field to fill to, or "Value (base field name)".

In addition, it would be great if I could specify a regex to select the other time series up to which Grafana should fill. If the regex doesn't match any, then do not fill, and if it would be ambigious then I'd be fine if it e.g. chooses the first matching in lexical order.

On the other hand, maybe I don't understand "Value (base field name)" well enough, but I couldn't find the documentation for that?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). 

I'm finding it a little hard to follow what you're asking for. Can you provide an example of your use case in an [issue](https://github.com/grafana/grafana/issues) to request the feature?
